server/serverless.yml

```diff
@@ -67,9 +68,31 @@ provider:
         - dynamodb:BatchWriteItem
         - dynamodb:BatchGetItem
       Resource:
-        - { "Fn::GetAtt": ["ContactsDynamoDBTable", "Arn" ] }
+        ## #Odavi's
+#        - { "Fn::GetAtt": ["ContactsDynamoDBTable", "Arn" ] }
         - { "Fn::GetAtt": ["ContactUpdatesDynamoDBTable", "Arn" ] }
         - { "Fn::GetAtt": ["GroupsDynamoDBTable", "Arn" ] }
+    -
```

Comment:
why we need another "ContactsDynamoDBTable" IAM rule? It seems like a copy of the previous one